### PR TITLE
Change name to Nim and fix Travis image link

### PR DESCRIPTION
**Pull request by clementi**
Fixes to the README, mainly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,4 +12,4 @@ Please see the [contributing guide](https://github.com/exercism/x-api/blob/maste
 
 The MIT License (MIT)
 
-Copyright (c) 2014 Katrina Owen, _@kytrinyx.com
+Copyright (c) 2014 Katrina Owen, @kytrinyx.com
```
Comment (kytrinyx) [inline README.md]: My email address is `_@kytrinyx.com`

**Comment by kytrinyx** [comment]
Good catch, thanks!

File: README.md (hunk 1 above)
Comment (clementi) [inline README.md]: @kytrinyx Whoops, my bad. Fixed in a new commit.

Comment (kytrinyx) [inline README.md]: Sweet, thanks :)

**Pull request merged**